src/main/scala/cromwell/webservice/CromwellApiHandler.scala

```diff
@@ -116,7 +116,8 @@ class CromwellApiHandler(workflowManager: ActorRef) extends Actor {
     case CallStdoutStderr(id, callFqn) =>
       val eventualCallLogs = ask(workflowManager, WorkflowManagerActor.CallStdoutStderr(id, callFqn)).mapTo[(WdlFile, WdlFile)]
```

Comment:
What about now passing around the tuple of WdlFiles but have a container object which had both?